core/store/orm/orm.go

```diff
@@ -419,7 +419,7 @@ func (orm *ORM) FindServiceAgreement(id string) (models.ServiceAgreement, error)
 // Jobs fetches all jobs.
 func (orm *ORM) Jobs(cb func(*models.JobSpec) bool, initrTypes ...string) error {
 	orm.MustEnsureAdvisoryLock()
-	return Batch(1000, func(offset, limit uint) (uint, error) {
+	return Batch(100, func(offset, limit uint) (uint, error) {
```

Comment:
This batch number should be defined as a constant and placed somewhere with a description.  It would improve discoverability and document why it is set at a certain value.